PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py

```diff
@@ -0,0 +1,205 @@
+    maxDPtRel   = cms.double(0.5),          # Minimum deltaPt/Pt for the match
+    resolveAmbiguities    = cms.bool(True), # Forbid two RECO objects to match to the same GEN object
+    resolveByMatchQuality = cms.bool(True), # False = just match input in order; True = pick lowest deltaR pair first
+) 
+
+lowPtElectronsMCMatchForTable = cms.EDProducer(
+    "MCMatcher",                            # cut on deltaR, deltaPt/Pt; pick best by deltaR
+    src         = lowPtElectronTable.src,   # final reco collection
+    matched     = cms.InputTag("finalGenParticles"), # final mc-truth particle collection
+    mcPdgId     = cms.vint32(11),           # one or more PDG ID (11 = ele); absolute values (see below)
+    checkCharge = cms.bool(False),          # True = require RECO and MC objects to have the same charge
+    mcStatus    = cms.vint32(1),            # PYTHIA status code (1 = stable, 2 = shower, 3 = hard scattering)
+    maxDeltaR   = cms.double(0.3),          # Minimum deltaR for the match
+    maxDPtRel   = cms.double(0.5),          # Minimum deltaPt/Pt for the match
+    resolveAmbiguities    = cms.bool(True), # Forbid two RECO objects to match to the same GEN object
+    resolveByMatchQuality = cms.bool(True), # False = just match input in order; True = pick lowest deltaR pair first
+)
+
+from PhysicsTools.NanoAOD.electrons_cff import electronMCTable
+lowPtElectronMCTable = cms.EDProducer(
+    "CandMCMatchTableProducer",
+    src = lowPtElectronTable.src,
+    mcMapDressedLep = cms.InputTag("lowPtElectronsMCMatchForTableAlt"),
+    mcMap = cms.InputTag("lowPtElectronsMCMatchForTable"),
+    mapTauAnc = cms.InputTag("matchingLowPtElecPhoton:hasTauAnc"),
+    objName = lowPtElectronTable.name,
+    objType = electronMCTable.objType,
+    branchName = cms.string("genPart"),
+    docString = cms.string("MC matching to status==1 electrons or photons"),
+    genparticles = cms.InputTag("finalGenParticles"), 
+)
+
+################################################################################
+# Sequences
+################################################################################
+
+lowPtElectronSequence = cms.Sequence(modifiedLowPtElectrons
+                                     +lowPtPATElectronID
+                                     +isoForLowPtEle
+                                     +updatedLowPtElectronsWithUserData
+                                     +finalLowPtElectrons)
+lowPtElectronTables = cms.Sequence(lowPtElectronTable)
+lowPtElectronMC = cms.Sequence(
+    particleLevelForMatchingLowPt
+    +tautaggerForMatchingLowPt
+    +matchingLowPtElecPhoton
+    +lowPtElectronsMCMatchForTable
+    +lowPtElectronsMCMatchForTableAlt
+    +lowPtElectronMCTable)
+
+################################################################################
+# Modifiers
+################################################################################
+
+_modifiers = ( run2_miniAOD_80XLegacy |
```

Comment:
@slava77 it's my understanding (after speaking with @mariadalfonso) that these modifiers are the only ones required to ensure the low pT ele are added to nanoAOD for `run2_nanoAOD_106Xv2` only.